templates/adguardhome/meta.yaml

```diff
@@ -0,0 +1,36 @@
+name: adguardhome
+description: adguard/adguardhome:0.107.16
+instructions: null
+changeLog:
+  - date: 2022-10-30
+    description: first release
+links:
+  - label: Documentation
+    url: https://github.com/AdguardTeam/AdGuardHome/wiki
+  - label: Github
+    url: https://github.com/AdguardTeam/AdGuardHome
+contributors:
+  - name: Supernova3339
+    url: https://github.com/Supernova3339
+schema:
+  type: object
+  required:
+    - projectName
+    - domain
+    - appServiceName
+    - appServiceImage
+  properties:
+    projectName:
+      type: string
+      title: Project Name
+    domain:
```

Comment:
The domain doesn't seem to be used, please remove it

Reply:
will do later, but some of them do actually use it

Reply:
removed